app/client-app/src/app/pages/Wordlist/index.jsx

```diff
@@ -13,12 +23,18 @@ import { WordCard } from "./components/WordCard";
 import WordlistSortingAndFilteringPanel from "./WordlistSortingAndFiltering";
 import { FixedSizeList } from "react-window";
 import AutoSizer from "react-virtualized-auto-sizer";
+import { FormControl, FormLabel } from "@chakra-ui/react";
 
 const Wordlist = () => {
   const [wordlist, setWordlist] = useState(null);
   const [cards, setCards] = useState([]);
   const { sorting, onSort, outputList, filter, setFilter } =
     useWordlistSortingAndFiltering(cards);
+  const [sliderValues, setSliderValues] = useState([1, 15]);
```

Comment:
Is it possible to set the max slider value based on the longest word in the database? There are certainly words longer than 15 characters